files/jquery.simpleselect/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "jquery.simpleselect",
+  "repo": "pioul/jQuery.SimpleSelect",
+  "files": {
+    "include": ["jquery.simpleselect.min.js"]
```

Comment:
you can use `*.min.*` to include all the min automatically. You are missing the css right now